packages/contract/src/contract-factory.ts

```diff
@@ -1,26 +1,33 @@
-import { Interface } from '@fuel-ts/abi-coder';
+import { Interface, WORD_SIZE } from '@fuel-ts/abi-coder';
 import type { JsonAbi, InputValue } from '@fuel-ts/abi-coder';
 import type {
   Account,
   CreateTransactionRequestLike,
   Provider,
+  TransactionRequest,
   TransactionResult,
   TransactionType,
 } from '@fuel-ts/account';
-import { CreateTransactionRequest } from '@fuel-ts/account';
+import {
+  CreateTransactionRequest,
+  BlobTransactionRequest,
+  TransactionStatus,
+  calculateGasFee,
+} from '@fuel-ts/account';
 import { randomBytes } from '@fuel-ts/crypto';
 import { ErrorCode, FuelError } from '@fuel-ts/errors';
+import { hash } from '@fuel-ts/hasher';
 import type { BytesLike } from '@fuel-ts/interfaces';
+import { bn } from '@fuel-ts/math';
 import { Contract } from '@fuel-ts/program';
 import type { StorageSlot } from '@fuel-ts/transactions';
 import { arrayify, isDefined } from '@fuel-ts/utils';
 
-import {
-  MAX_CONTRACT_SIZE,
-  getContractId,
-  getContractStorageRoot,
-  hexlifyWithPrefix,
-} from './util';
+import { getLoaderInstructions, getContractChunks } from './loader';
+import { getContractId, getContractStorageRoot, hexlifyWithPrefix } from './util';
+
+/** Amount of percentage override for chunk sizes in blob transactions */
+export const CHUNK_SIZE_OVERRIDE = 0.05;
```

Comment:
```suggestion
const CHUNK_SIZE_OVERRIDE = 0.05;
```

IMO, we don't need to export this constant.